docs/configuration.md

```diff
@@ -146,6 +147,22 @@ For example, if you have 10 ETH available in your wallet on the exchange and `tr
 !!! Warning
     `tradable_balance_ratio` applies to the current balance (free balance + tied up in trades). Therefore, assuming a starting balance of 1000, a configuration of `tradable_balance_ratio=0.99` will not guarantee that 10 units will always remain available on the exchange. The free amount may reduce to 5 units if the total balance is reduce to 500 (either by a losing streak, or by withdrawing balance).
 
+#### Amend last stake amount
+
+Assuming we have a `balance=1000` (USDT), `stake_amount=400`, and `max_open_trades=3`.
```

Comment:
"we have the tradable balance of 1000 USDT, ..."
-- that's since we should count here the `tradable_balance_ratio` for balance. But mentioning this option would make things more complicated than it should be, so a sentence like this can be used.